services/api_gateway/pom.xml

```diff
@@ -20,8 +20,8 @@
 
     <parent>
         <groupId>org.springframework.boot</groupId>
-        <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.1.6.RELEASE</version>
+            <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.3.5.RELEASE</version>
```

Comment:
we could go back to the previous version of springframework

Reply:
Correct. Roll back to previous version of Spring Boot and Cloud GW.